recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "tudatpy" %}
   {% set version = "0.8.0.dev9" %}
   {% set build = "0" %}
-  {% set git_rev = "v0.8.0.dev9" %}
+  {% set git_rev = "c51a1b5490df82bec17c3a12108c3f304412a539" %}
```

Comment:
Why this specific commit?

Reply:
I had a look and this was a very old version, which may explain some of the compilation errors. I've updated it